src/services/document-service.spec.js

```diff
@@ -817,9 +817,10 @@ describe('document-service', () => {
         language: 'en'
       });
 
+      // This archived document should never be matched:
```

Comment:
Could we turn this comment into an assert? 🙏

Reply:
True, that would be better, will do

Reply:
Done!